script.js

```diff
@@ -151,3 +151,23 @@ submit.addEventListener('click', (event) => {
   }
   return true;
 });
+
+// PRESERVE DATA //////////////////////////////////////////////////////////////////////////////////
+const userInput = document.querySelectorAll('.userData');
+const form = document.querySelector('.form');
+let data = {};
+
+if (localStorage.getItem('userData')) {
+  data = JSON.parse(localStorage.getItem('userData'));
+  userInput.forEach((input) => {
+    input.value = data[input.name];
+  });
+}
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  userInput.forEach((input) => {
+    data[input.name] = input.value;
+  });
+  localStorage.setItem('userData', JSON.stringify(data));
+});
```

Comment:
Great effort so far. However, I can see a few things you need to update in order to match the project requirements. 

- [ ] Currently, the data is not saved to the local storage unless you click on the `Get in Touch` button hence the submit button does not work even if the inputs are correct which is not the best way. Kindly ensure that the submit button works correctly and the data is saved into the local storage when they are typed in the input fields without clicking any button. 
- [ ] If at the previous visit to the page, in the same browser, no input was entered in the form then the form inputs should remain empty. However, I see that the data in the localstorage persists and it is removed only when you clear them using the clear all button in the browser console. 

Kindly check and fix this issues to make your project one of the best and up to standard. :pray: